tools/osv-linter/internal/checks/packages.go

```diff
@@ -39,6 +40,9 @@ func PackageExists(json *gjson.Result, config *Config) (findings []CheckError) {
 		// Normalize ecosystems with a colon to their base.
 		// e.g. "Alpine:v3.5" -> "Alpine"
 		ecosystem := strings.Split(value.Get("package.ecosystem").String(), ":")[0]
+		if config.Ecosystems != nil && !slices.Contains(config.Ecosystems, ecosystem) {
```

Comment:
I think `nil` is a valid slice value in Go? Which would imply that `slices.Contains` will just work on `nil` slice values and that we can just remove the `nil` check. 

Can you please check this?

Reply:
This is an allowlist of ecosystems (when specified). Without ensuring that `config.Ecosystems` is actually set to something, `slices.Contains` does indeed return `true`, but that's not the desired behaviour. Added some explanatory comments in [5fd8b08](https://github.com/ossf/osv-schema/pull/285/commits/5fd8b080fb8b4b129abce9a89fa904bef7a2a0b2)

Reply:
Ack